scripts/intel_study_scripts/README.md

```diff
@@ -0,0 +1,53 @@
+**Instructions to build Galois and reproduce IntelStudy experiments**
+
+#### Clone the repository
+git clone https://github.com/IntelligentSoftwareSystems/Galois
+
+##Let us assume that the SRC_DIR is the top-level Galois source dir where the Galois repository is cloned.
+
+#### Building Galois
+BUILD_DIR=<path-to-your-build-dir>
+
+mkdir -p $BUILD_DIR
+cmake -S $SRC_DIR -B $BUILD_DIR -DCMAKE_BUILD_TYPE=Release
+
+##Galois applications are in lonestar directory. In order to build a particular application:
+make -C $BUILD_DIR/lonestar/analytics/cpu/<app-dir-name> -j
+
+## For IntelStudy build the following apps:
+#BFS
+cd $BUILD_DIR/lonestar/analytics/cpu/bfs && make -j
```

Comment:
This is fine but for consistency, you can change all these app builds to: "make -C <directory> -j"